### PR TITLE
Move periodic-imagestream-importer to app.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
follow up https://github.com/openshift/release/pull/14507#issuecomment-749922746

We update the import on app.ci and if the image is used by any test, it will be distributed to other clusters too (by test-images-distributor).

/cc @alvaroaleman

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

wait for the rehearsal.

**Review by alvaroaleman — APPROVED** [review]

**Comment by hongkailiu** [comment]
On app.ci (rehearsal):
```
Failed to pull image "registry.svc.ci.openshift.org/ocp/4.7:cli": rpc error: code = Unknown desc = Error reading manifest cli in registry.svc.ci.openshift.org/ocp/4.7: unauthorized: authentication required
```
This is what I was expecting on api.ci too.

I think the reason is

```
oc get secret -n openshift-config pull-secret -o yaml | yq -r '.data.".dockerconfigjson"' | base64 -d | jq -r '.auths|keys[]'
cloud.openshift.com
quay.io
registry.connect.redhat.com
registry.redhat.io
```
Let me use the registry on app.ci too.

**Comment by hongkailiu** [comment]
Not cool.

```
Failed to pull image "registry.ci.openshift.org/ocp/4.7:cli": rpc error: code = Unknown desc = Error reading manifest cli in registry.ci.openshift.org/ocp/4.7: unauthorized: authentication required
```

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1179,7 +1179,7 @@ periodics:
         done
       command:
       - /bin/bash
-      image: registry.svc.ci.openshift.org/ocp/4.7:cli
+      image: image-registry.openshift-image-registry.svc:5000/ocp/4.7:cli
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-periodics.yaml:1182]: This is not nice.
The public one looks much nicer.

Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml:1182]: Asking registry team:
https://coreos.slack.com/archives/C013VBYBJQH/p1608755279429100

Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml:1182]: Had to do this like this for now.
/hold cancel

**Review by alvaroaleman — APPROVED** [review]

**Comment by hongkailiu** [comment]
Works on app.ci 
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-imagestream-importer/1342166177626984448

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml:1182]: https://bugzilla.redhat.com/show_bug.cgi?id=1911470

**Pull request merged**